python/GafferImageTest/ResampleTest.py

```diff
@@ -124,12 +125,13 @@ def __test( fileName, size, filter ) :
 			# of the current directory.
 			if False :
 
-				if not os.path.exists( "resampleComparison" ) :
-					os.makedirs( "resampleComparison" )
+				resampleComparisonDir = pathlib.Path( "resampleComparison" )
+				if not resampleComparisonDir.exists() :
+					resampleComparisonDir.mkdir()
```

Comment:
Not important here, but can be in other situations : `if not exists then mkdir` is in a race with other processes doing the same thing, and can lead to a process throwing an exception if the directory is created in between it checking and trying to create. `path.mkdir( exist_ok=True )` is a one-liner without this problem.